ciri/api/http/http.c

```diff
@@ -28,14 +28,17 @@ typedef struct iota_api_http_session_s {
 } iota_api_http_session_t;
 
 static retcode_t error_serialize_response(iota_api_http_t *const http, error_res_t **const error,
-                                          char const *const message, char_buffer_t *const out) {
+                                          char const *const message, uint16_t const status_code,
+                                          char_buffer_t *const out) {
   retcode_t ret = RC_OK;
 
   if (*error == NULL) {
-    if (message == NULL) {
+    if (message == NULL && status_code == MHD_HTTP_INTERNAL_SERVER_ERROR) {
```

Comment:
Why this `&&` ?

Reply:
I originally thought I should use `Bad request` as dfault error status code.
I will change `&&` to `||`

Reply:
I don't think it's necessary at all

Reply:
what sould I do?

Reply:
remove `message == NULL`？

Reply:
Remove `&& status_code == MHD_HTTP_INTERNAL_SERVER_ERROR`

Reply:
OK